1. Desktop app calls Web API/TodoListClient/MainWindow.xaml.cs

```diff
@@ -62,6 +62,15 @@ public partial class MainWindow : Window
         private static readonly string TodoListScope = ConfigurationManager.AppSettings["todo:TodoListScope"];
         private static readonly string TodoListBaseAddress = ConfigurationManager.AppSettings["todo:TodoListBaseAddress"];
         private static readonly string[] Scopes = { TodoListScope };
+        private static string TodoListApiAddress
+        {
+            get
+            {
+                string baseAddress = TodoListBaseAddress;
+                return baseAddress.EndsWith("/") ? TodoListBaseAddress + "api/todolist"
```

Comment:
Adding a bit of flexibility (allowing the TodolistbaseAddress to end or not with a slash